src/components/AccountCard.tsx

```
@@ -3,36 +3,19 @@ import { useState, useEffect } from "react";
-  cardData: CardData[];
-  selectedCard: CardData | undefined;
-  setSelectedCard: React.Dispatch<React.SetStateAction<CardData | undefined>>;
+  cardData: any;
+  data:any
 }
 
 const AccountCard: React.FC<SourceAccountProps> = ({
   cardData,
-  selectedCard,
-  // setSelectedCard,
+  data
```

ESLint

[error] 'data' is defined but never used.

(@typescript-eslint/no-unused-vars)